CONTRIBUTING.md

```diff
@@ -1,79 +1,81 @@
-You came to the right place to start your contribution, follow the guidlines and let us know if we can help with anything else.
-
-## Grommet projects
-
-Grommet is divided into a several projects; the following are notable:
-
-- [grommet](https://github.com/grommet/grommet) - the primary
-  Grommet 2.X project is actively developped and contributing is more than welcome! Be sure to check the [good first issues](https://github.com/grommet/grommet/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22)
-- [grommet-icons](https://github.com/grommet/grommet-icons) -
-  Iconography for Grommet and React.js
-- [react-desc](https://github.com/grommet/react-desc)
-  Add a schema to your React components based on React PropTypes
-- [design-kit](https://github.com/grommet/design-kit)
-  The Grommet Design Kit provides a set of sticker sheets and templates to help bootstrap your design process.
-
-## YOU can Become a Contributor
-
-Quick steps and ideas of how you can become a contributor!
-
-1.  Code Code Code (and file Pull Request)
-2.  Create design assets or style guide revsions
-3.  Submit updates and improvements to the documentation.
-4.  Submit articles and guides which are also part of the documentation.
-5.  Join the Slack to help and interact with grommet users
-6.  Help a Grommet designer or developer by answering questions on
-    StackOverflow, Slack and GitHub.
-7.  Report bugs and proposing new features by filing issues on Github or ask in our Slack community
-8.  Share with us exciting projects using grommet on Slack
-
-## Contributing
+Thank you for reaching the contribution page and showing the true gremlin in
+you! In Grommet we do believe that the more the merrier – Welcome! Thank you for
+making this step of joining and contributing to our community, and for helping
+us make Grommet the best tool for streamlining the way you develop apps. You
+came to the right place to start your contribution! Follow the guidelines and
+let us know if we can help with anything else.
+
+## Grommet Projects
+
+Grommet is divided into several projects, the following are notable:
+
+- [grommet] – the primary Grommet 2.X project is actively developed and
+  contributions are more than welcome! Be sure to check the [good first issues].
+- [grommet-icons] – iconography for Grommet and React.js.
+- [react-desc] – add a schema to your React components based on React
+  [`PropTypes`][prop-types].
+- [design-kit] – the Grommet Design Kit provides a set of sticker sheets and
+  templates to help bootstrap your design process.
+
+## You can Become a Contributor
+
+Afterall, that’s why you’re here, right?
+Quick steps and ideas of how you can contribute to Grommet:
+
+1. Code, code, code… (and make a Pull Request).
+1. Create design assets or style guide revisions.
```

Comment:
The bullet numbers should be consecutive 1-8

Reply:
Markdown renders them that way. If we force the consecutive ordering in the source of the file as well, it is more difficult to insert new items partway through the list:

```markdown
1. Code, code, code…
1. Create design assets
1. Do something else
1. Another thing
```

gets compiled to:

```html
<ol>
  <li>Code, code, code…</li>
  <li>Create design assets</li>
  <li>Do something else</li>
  <li>Another thing</li>
</ol>
```

which gets rendered as:

```text
1. Code, code, code…
2. Create design assets
3. Do something else
4. Another thing
```

So then if we add a new item right in the middle later, we don’t have to worry about adjusting all of the numbers for each item after it in the list. We’re essentially just cueing Markdown that, “Hey, this is an _ordered_ list.”